src/prefect/environments/storage/_healthcheck.py

```diff
@@ -122,12 +122,12 @@ def environment_dependency_check(flows: list):
         for dependency in required_imports:
             try:
                 importlib.import_module(dependency)
-            except ModuleNotFoundError:
+            except ModuleNotFoundError as moduleNotFoundError:
```

Comment:
There's no need to give these long names, `exc` is common here and should be fine.

```suggestion
            except ModuleNotFoundError as exc:
```

Reply:
thanks @jcrist for the suggestion will try to keep that going forward as a practise